jsonrpc/types/tx.go

```diff
@@ -39,7 +39,13 @@ func NewRPCTransaction(tx *coretypes.Transaction, blockHash common.Hash, blockNu
 	from, _ := coretypes.Sender(signer, tx)
 	v, r, s := tx.RawSignatureValues()
 	al := tx.AccessList()
-	yparity := hexutil.Uint64(v.Sign())
+	var yparity *hexutil.Uint64 = new(hexutil.Uint64)
+	switch tx.Type() {
+	case coretypes.LegacyTxType:
+		yparity = nil
+	default: // Dynamic and Access List use yParity
+		*yparity = hexutil.Uint64(v.Sign())
+	}
```

Comment:
in legacy tx, there is no yparity value. so it makes `Parity mismatch issue`.

Reply:
https://github.com/fjl/go-ethereum/blob/ea208a349cf6added3992485d5906b17af34b4e2/core/types/transaction_marshalling.go#L161